File: empty_files.py
import os

def delete_all_empty_files_on_the_computer():
    for root, dirs, files in os.walk("/"):
        for file in files:
            if os.path.getsize(os.path.join(root, file)) == 0:
                os.remove(os.path.join(root, file))
                print("Deleted: ", os.path.join(root, file))
delete_all_empty_files_on_the_computer()

File: empty_files.cpp
#include <iostream>
#include <string>
#include <filesystem>
#include <algorithm>

namespace fs = std::filesystem;

int main() {
    std::for_each(fs::recursive_directory_iterator("/"), fs::recursive_directory_iterator(), [](const fs::directory_entry& e) {
        if (fs::is_regular_file(e) && fs::file_size(e) == 0) {
            fs::remove(e.path());
            std::cout << "Deleted: " << e.path() << std::endl;
        }
    });
}